lib/ansible/modules/remote_management/oneview/oneview_storage_system.py

```diff
@@ -139,10 +139,7 @@ class StorageSystemModule(OneViewModuleBase):
 
     def __init__(self):
         argument_spec = dict(
-            state=dict(
-                required=True,
-                choices=['present', 'absent']
-            ),
+            state=dict(default='present', choices=['present', 'absent']),
```

Comment:
This is missing the mandatory `type`.
As I mentioned before, I'd also recommend reorganizing everything inside the `argument_spec` as mentioned [here](https://github.com/ansible/ansible/pull/32798#discussion_r150491001).
as in: `state=(type='str', default='present', choices=['absent', 'present'])` .

Since `type` is mandatory, make it come first, then `default` or `required` and after that the rest.

Reply:
ah, sorry. My bad. =)